eng/scripts/SetTestPipelineVersion.ps1

```diff
@@ -1,28 +1,18 @@
-# Overides the project file and CHANGELOG.md for the template project using the next publishable version
+# Overides the project file and CHANGELOG.md for the template project using a custom preview version
 # This is to help with testing the release pipeline.
-. "${PSScriptRoot}\..\common\scripts\common.ps1"
-$latestTags = git tag -l "Azure.Template_*"
-$semVars = @()
-
-$changeLogFile = "${PSScriptRoot}\..\..\sdk\template\Azure.Template\CHANGELOG.md"
 
-Foreach ($tags in $latestTags)
-{
-  $semVars += $tags.Replace("Azure.Template_", "")
-}
+[CmdletBinding(SupportsShouldProcess = $true)]
+param(
+  [Parameter(Mandatory = $true)]
+  [string]$PreviewVersionNumber
+)
 
-$semVarsSorted = [AzureEngSemanticVersion]::SortVersionStrings($semVars)
-LogDebug "Last Published Version $($semVarsSorted[0])"
+. "${PSScriptRoot}\..\common\scripts\common.ps1"
 
-$newVersion = [AzureEngSemanticVersion]::ParseVersionString($semVarsSorted[0])
-$newVersion.IncrementAndSetToPrerelease()
-LogDebug "Version to publish [ $($newVersion.ToString()) ]"
+$changeLogFile = "${PSScriptRoot}\..\..\sdk\template\Azure.Template\CHANGELOG.md"
+$newVersion = "1.0.3-beta.$PreviewVersionNumber"
```

Comment:
Why are we hardcoding `1.0.3`?

Reply:
We should be able to use your package parsing methods to get the version and the ChangeLog path.